src/poliastro/twobody/orbit.py

```diff
@@ -1497,3 +1504,294 @@ def plot(self, label=None, use_3d=False, interactive=False):
             from poliastro.plotting.core import OrbitPlotter2D
 
             return OrbitPlotter2D().plot(self, label=label)
+
+    @classmethod
+    def __groundtrack_orbit(
+        cls,
+        attractor,
+        Q,
+        i=None,
+        e=None,
```

Comment:
Should we call these `inc` and `ecc` for consistency?

Reply:
To clarify: perhaps we don't need to change the inner functions, but at least the `Orbit` interface should retain `inc` and `ecc`, I think.

Reply:
That sounds fine, I will make these changes.